sidecar/sokuji_sidecar/voxtral_stream.py

```
@@ -0,0 +1,170 @@
+        first = self._proc(first_audio, is_streaming=True, is_first_audio_chunk=True,
+                           return_tensors="pt").to(self._device, dtype=self._dtype)
+        streamer = TextIteratorStreamer(self._proc.tokenizer, skip_special_tokens=True,
+                                        clean_up_tokenization_spaces=True)
+
+        def _run():
+            try:
+                self._model.generate(
+                    input_ids=first.input_ids,
+                    input_features=self._input_features_generator(first.input_features),
+                    num_delay_tokens=first.num_delay_tokens, streamer=streamer)
+            except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)